x-pack/plugins/security_solution/public/management/components/artifact_list_page/hooks/use_artifact_card_props_provider.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { useCallback, useMemo } from 'react';
+import {
+  AnyArtifact,
+  ArtifactEntryCardProps,
+  useEndpointPoliciesToArtifactPolicies,
+} from '../../artifact_entry_card';
+import { useTestIdGenerator } from '../../hooks/use_test_id_generator';
+import { useGetEndpointSpecificPolicies } from '../../../services/policies/hooks';
+import { getLoadPoliciesError } from '../../../common/translations';
+import { useToasts } from '../../../../common/lib/kibana';
+
+type CardActionType = 'edit' | 'delete';
+
+export interface UseArtifactCardPropsProviderProps {
+  items: ExceptionListItemSchema[];
+  onAction: (action: { type: CardActionType; item: ExceptionListItemSchema }) => void;
+  cardActionEditLabel: string;
+  cardActionDeleteLabel: string;
+  dataTestSubj?: string;
+}
+
+export type ArtifactCardPropsProvider = (
```

Comment:
No need to export this. Not used anywhere else.

Reply:
👍 - done